### PR TITLE
Changing turfs no longer changes the wetness of the turf

### PR DESCRIPTION
**Pull request by ChangelingRain**
Crowbarring a lubed turf up will not magically eliminate the lube. The Lube Remains.
Iced turfs will be removed, but the removed tile or equivalent will be covered in ice and will place an iced turf.

### CODE REVIEW

**Comment by optimumtact** [comment]
Oh come on, this is obviously a balance change

**Comment by Iamgoofball** [comment]
# this removes crowbarring the ice off of tiles with freon

this is not a bug

i repeat

# not a bug

fuckyoujoan

**Comment by Iamgoofball** [comment]
@Cheridan 
I demand this PR be closed based on Joan outright admitting "yeah this was a feature i'm going to decide it's a bug now"

**Comment by lzimann** [comment]
@KorPhaeron You decide, fix or balance change?

**Comment by GunHog** [comment]
This does not make logical sense, though! If I crowbar up a wet tile, water/lube magically transfers to the plating under it? That would not happen unless the there is enough liquid to spill into the exposed plating!

**Comment by Iamgoofball** [comment]
This isn't a fix @lzimann 
 This is a feature removal. See the linked image for proof.

**Comment by Iamgoofball** [comment]
this also

# buffs freon

a lot

**Comment by Qustinnus** [comment]
seems dumb and not really like a fix more like a balance change so it's against the freeze.

**Comment by kevinz000** [comment]
wait so if i pour 300 units of lube on tiles they're going to be lubed for 20 minutes with no way to unlube 
hmmmmmmmmmmmmmm!

**Comment by Qustinnus** [comment]
max lube time is 300 seconds IIRC but still true

**Comment by Cyberboss** [comment]
This is still a bad idea imho

**Comment by RemieRichards** [comment]
This is dumb

File: code/game/turfs/simulated/floor.dm [hunk 1]
```diff
@@ -166,7 +166,9 @@
 		if(user && !silent)
 			to_chat(user, "<span class='danger'>You remove the floor tile.</span>")
 		if(floor_tile && make_tile)
-			new floor_tile(src)
+			var/obj/item/I = new floor_tile(src)
```
Comment (RemieRichards) [inline code/game/turfs/simulated/floor.dm]: these parts are the only good parts.
if you make a new pr with just the "crowbarring up ice remove it from the turf and makes ice floor tiles" parts (+ making it so if you lay that floor tile the new turf inherits the ice) then that'd probably be accepted for ~rule of cool~

**Comment by KorPhaeron** [comment]
What Remie said

**Pull request closed without merging**